lib/iris/cube.py

```diff
@@ -64,6 +64,8 @@ class _CubeFilter:
     """A constraint, paired with a list of cubes matching that constraint."""
 
     def __init__(self, constraint, cubes=None):
+        from iris.cube import CubeList
```

Comment:
Why are we importing from `iris.cube` while within the `iris.cube` module?

Reply:
Oof 🤦 
Will fix !